my_2d_robot/src/my_2d_robo_state_publisher.cpp

```diff
@@ -22,8 +25,9 @@ int main(int argc, char** argv)
     // There is one message per update. This is 2 since 2 joints are updated at a time.
     // joint_state.name = { "joint1", "joint2" };
 
-    float joint1_pos = ((rand() % 315) / 100.0);
-    float joint2_pos = ((rand() % 315) / 100.0);
+    // Randomly generating values in the range [0,2*PI].
+    float joint1_pos = ((rand() % (2 * 315)) / 100.0);
+    float joint2_pos = ((rand() % (2 * 315)) / 100.0);
 
     // The position is the angle of rotation.
```

Comment:
Remove this comment since the position (x, y) is actually not the same as the angle of rotation (radians).